package-lock.json

```diff
@@ -0,0 +1,6142 @@
+{
+  "name": "social-media-mockbee-template",
```

Comment:
Since we're using yarn in the initial template, wouldn't want to add npm to the template now. So use yarn instead and raise again

Reply:
Okay.